packages/core/src/Serialization/Update/PoolVotingThresholds.ts

```diff
@@ -17,6 +17,7 @@ export class PoolVotingThresholds {
   #committeeNormal: UnitInterval;
   #committeeNoConfidence: UnitInterval;
   #hardForkInitiation: UnitInterval;
+  #securityRelevantParamVotingThreshold: UnitInterval;
```

Comment:
```suggestion
  #securityRelevantParam: UnitInterval;
```

I would drop the `VotingThreshold`  posfix as these are all voting thresholds

Reply:
Done here: [69c9f25](https://github.com/input-output-hk/cardano-js-sdk/pull/1327/commits/69c9f25ee0e473f22994216696e3551361b4affd)